content/cli/workspaces.md

```diff
@@ -0,0 +1,206 @@
+### Workspaces
+
+Nest has two modes for organizing code:
+- **single project mode**: useful for building individual project-focused applications that have their own dependencies and don't need to optimize for sharing modules, or optimizing complex builds.  This is the default mode.
```

Comment:
Not sure about the naming here (**single project mode**), but maybe I'm overreacting :D

Reply:
Changing to "standard mode" and "monorepo mode" in next draft.